1559699329133	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1559701255183	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559701255327	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559701255327	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559701255328	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559701255328	Sync.Service	INFO	Starting sync at 2019-06-05 04:20:55 in browser session DLDB2kz0MYXZ
1559701255328	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1559701255332	Sync.Status	INFO	Resetting Status.
1559701255332	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1559701255333	Sync.BrowserIDManager	INFO	Getting keys
1559701255333	Sync.BrowserIDManager	INFO	Getting an assertion from: https://token.services.mozilla.com/1.0/sync/1.5
1559701255333	FirefoxAccounts	DEBUG	enter getAssertion()
1559701255333	FirefoxAccounts	DEBUG	getKeypairAndCertificate: already have keyPair and certificate
1559701255333	FirefoxAccounts	DEBUG	getAssertionFromCert
1559701255337	FirefoxAccounts	DEBUG	getAssertionFromCert returning signed: true
1559701255338	Sync.BrowserIDManager	DEBUG	Getting a token
1559701255338	Services.Common.TokenServerClient	DEBUG	Beginning BID assertion exchange: https://token.services.mozilla.com/1.0/sync/1.5
1559701255338	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1559701255338	Services.Common.RESTRequest	TRACE	HTTP Header accept: application/json
1559701255338	Services.Common.RESTRequest	TRACE	HTTP Header authorization: ***** (suppressed)
1559701255338	Services.Common.RESTRequest	TRACE	HTTP Header x-client-state: ***** (suppressed)
1559701255755	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559701255839	Services.Common.RESTRequest	TRACE	onStartRequest: GET https://token.services.mozilla.com/1.0/sync/1.5
1559701255839	Services.Common.RESTRequest	TRACE	Channel for GET https://token.services.mozilla.com/1.0/sync/1.5 returned status code 2152398878
1559701255839	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1559701255839	Services.Common.RESTRequest	TRACE	GET body: 
1559701255840	Sync.BrowserIDManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.js:35:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.js:58:16
getTokenFromBrowserIDAssertion@tokenserverclient.js:244:13
1559701255840	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1559701255840	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1559701255840	Sync.BrowserIDManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.js:35:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.js:58:16
getTokenFromBrowserIDAssertion@tokenserverclient.js:244:13
1559701255843	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1559701255843	FirefoxAccounts	DEBUG	getOAuthToken enter
1559701255845	FirefoxAccounts	TRACE	getCachedToken returning cached token
1559701255845	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1559701255846	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1559701255846	Services.Common.RESTRequest	TRACE	HTTP Header authorization: ***** (suppressed)
1559701255846	Services.Common.RESTRequest	TRACE	HTTP Header accept: application/json
1559701255846	Services.Common.RESTRequest	TRACE	HTTP Header if-none-match: "1d3cb6698a7af7b246a9837b970c1e724afce599-gzip"
1559701256013	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1559701256013	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1559701256014	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1559701256721	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559701256826	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559701256826	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559701256827	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559701256827	Sync.Service	INFO	Starting sync at 2019-06-05 04:20:56 in browser session DLDB2kz0MYXZ
1559701256831	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:41:5) JS Stack trace: LockException@util.js:41:5
throwLockException@util.js:115:11
WrappedLock@util.js:129:15
_lockedSync@service.js:1164:8
sync/<@service.js:1118:18
WrappedCatch@util.js:103:27
sync@service.js:1119:7
1559701256831	Sync.Service	INFO	Cannot start sync: already syncing?
1559701258759	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559701307532	Services.Common.RESTRequest	TRACE	onStartRequest: GET https://profile.accounts.firefox.com/v1/profile
1559701307532	Services.Common.RESTRequest	TRACE	Channel for GET https://profile.accounts.firefox.com/v1/profile returned status code 2152398862
1559701307532	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398862 - NS_ERROR_NET_TIMEOUT
1559701307532	Services.Common.RESTRequest	TRACE	GET body: 
1559701307532	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_NET_TIMEOUT\"  nsresult: \"0x804b000e (NS_ERROR_NET_TIMEOUT)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]"}
1559703165154	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559703165295	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559703165295	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559703165296	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559703165296	Sync.Service	INFO	Starting sync at 2019-06-05 04:52:45 in browser session DLDB2kz0MYXZ
1559703165296	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:41:5) JS Stack trace: LockException@util.js:41:5
throwLockException@util.js:115:11
WrappedLock@util.js:129:15
_lockedSync@service.js:1164:8
sync/<@service.js:1118:18
WrappedCatch@util.js:103:27
sync@service.js:1119:7
1559703165296	Sync.Service	INFO	Cannot start sync: already syncing?
1559703165776	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559703166188	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1559703166188	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1559703166189	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1559703166702	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559703167169	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559703167170	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559703167171	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559703167172	Sync.Service	INFO	Starting sync at 2019-06-05 04:52:47 in browser session DLDB2kz0MYXZ
1559703167178	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:41:5) JS Stack trace: LockException@util.js:41:5
throwLockException@util.js:115:11
WrappedLock@util.js:129:15
_lockedSync@service.js:1164:8
sync/<@service.js:1118:18
WrappedCatch@util.js:103:27
sync@service.js:1119:7
1559703167178	Sync.Service	INFO	Cannot start sync: already syncing?
1559703168091	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559703174224	Sync.Resource	WARN	GET request to https://sync-535-us-west-2.sync.services.mozilla.com/1.5/80856074/info/collections failed: [Exception... "The connection has timed out"  nsresult: "0x804b000e (NS_ERROR_NET_TIMEOUT)"  location: "<unknown>"  data: no] No traceback available
1559703174225	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1559703174225	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1559703174225	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The connection has timed out"  nsresult: "0x804b000e (NS_ERROR_NET_TIMEOUT)"  location: "<unknown>"  data: no] No traceback available
1559703174226	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1559703174230	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The connection has timed out"  nsresult: "0x804b000e (NS_ERROR_NET_TIMEOUT)"  location: "<unknown>"  data: no] No traceback available
